### PR TITLE
README: SQL Server SKU name to SqlServerVersion enum

### LINKED ISSUE #86: SqlServerVersion
I find it difficult to figure out what to put as the value for `<SqlServerVersion>` in the project file. 

Let's assume I want to target `SQL Server 2016`.

First, I can map `SQL Server 2016` to `13.0` using this page:
https://support.microsoft.com/en-us/help/321185/how-to-determine-the-version-edition-and-update-level-of-sql-server-an 

Next, I can map `13.0` to `Sql130` using this page:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.sqlserver.dac.model.sqlserverversion

Perhaps the docs and `dotnet template` could include these links. Maybe there's a more direct way to figure this out?

Let me know what you think would be a good resolution :-)

### PR DESCRIPTION
**Pull request by brettrowberry**
Explains how to go from the SQL Server SKU name to the SqlServerVersion enum.

Resolves #86

I broke #88 by deleting my fork!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,11 @@ The simplest way to get started is to install our templates with `dotnet new` us
 dotnet new --install MSBuild.Sdk.SqlProj.Templates
 ```
 
-You can then create a new project file using the following command. If you don't want to target the latest version of SQL Server you can specify a version to target using the `-s Sql<version>` switch. Please refer to [this document](https://docs.microsoft.com/dotnet/api/microsoft.sqlserver.dac.model.sqlserverversion) for the available versions.
+You can then create a new project file using the following command. If you don't want to target the latest version of SQL Server you can specify a version to target using the `-s Sql<version>` switch. 
+
+See [How to determine the version, edition, and update level of SQL Server and its components](https://support.microsoft.com/en-us/help/321185/how-to-determine-the-version-edition-and-update-level-of-sql-server-an) to map from the SQL Server SKU name to the version number, e.g. `SQL Server 2019` to `15.0`.
+
+See [SqlServerVersion Enum](https://docs.microsoft.com/en-us/dotnet/api/microsoft.sqlserver.dac.model.sqlserverversion) to map from the SQL Server version number to the `SqlServerVersion` needed by the template and project file, e.g. `15.0` to `Sql150`.
```
Comment (ErikEJ) [inline README.md:28]: Remove /en-us

Comment (ErikEJ) [inline README.md:28]: SQL Server SKU ??

**Review by ErikEJ — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (brettrowberry) [reply README.md]: I'm guessing you mean replace `SQL Server SKU name` with `SQL Server SKU`.

Comment (ErikEJ) [reply README.md]: Sorry, got confused, better keep it as SKU name

**Review by jmezach — APPROVED** [review]

**Review by jmezach — APPROVED** [review]

**Comment by jmezach** [comment]
LGTM

**Comment by brettrowberry** [comment]
Sorry @jmezach I accidentally dismissed your review.

**Review by ErikEJ — CHANGES_REQUESTED** [review]

**Review by jmezach — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -21,10 +21,19 @@ The simplest way to get started is to install our templates with `dotnet new` us
...
+
+If you don't want to target the latest version of SQL Server, you can specify a version to target using the `-s Sql<version>` switch. 
+
+See [How to determine the version, edition, and update level of SQL Server and its components](https://support.microsoft.com/help/321185/how-to-determine-the-version-edition-and-update-level-of-sql-server-an) to map from the SQL Server SKU name to the version number, e.g. `SQL Server 2019` to `15.0`.
+
+See [SqlServerVersion Enum](https://docs.microsoft.com/dotnet/api/microsoft.sqlserver.dac.model.sqlserverversion) to map from the SQL Server version number to the `SqlServerVersion` needed by the template and project file, e.g. `15.0` to `Sql150`.
 
 ```
-dotnet new sqlproj [-s Sql150]
+dotnet new sqlproj -s Sql150
```
[the remark this answers is not in the document]
Comment (brettrowberry) [reply README.md]: Good find. I changed it to 130.

**Review by ErikEJ — APPROVED** [review]

**Pull request merged**